tests/test_runs/test_run_functions.py

```diff
@@ -374,6 +388,23 @@ def test_run_and_upload_randomsearch(self):
         # it has a different value than the other examples before
         self._run_and_upload(randomsearch, '12172')
 
+    def test_run_and_upload_maskedarrays(self):
+        # This testcase is important for 2 reasons:
+        # 1) it verifies the correct handling of masked arrays (not all parameters are active)
+        # 2) it verifies the correct handling of a 2-layered grid search
+        gridsearch = GridSearchCV(
```

Comment:
Could you please explain what makes this example conditional? Right now it looks like a shorter version of the example above and I don't see where the conditionality comes in.

Reply:
I added "Note that this is a list of dictionaries, all containing 1 hyperparameter. " I assume you agree